chain/network/src/peer.rs

```diff
@@ -641,6 +659,13 @@ impl StreamHandler<Result<Vec<u8>, ReasonForBan>> for Peer {
                 return;
             }
         };
+        if let PeerMessage::Routed(rm) = &peer_msg {
```

Comment:
why not `if let PeerMessage::Routed(RoutedMessageBody::ForwardTx(_)) = &peer_msg`?

Reply:
Done in https://github.com/nearprotocol/nearcore/pull/3317/commits/ddacef7f99d43a7130dc704c27f13242bc01016c